src/cpp/tests/testthat/themes/rsthemes/zenburnesque.rstheme

```diff
@@ -101,20 +107,20 @@
 .ace_marker-layer .ace_foreign_line {
   position: absolute;
   z-index: -1;
-  background-color: #4F4F4F;
+  background-color: #505050;
 }
 .ace_marker-layer .ace_active_debug_line {
   position: absolute;
   z-index: -1;
-  background-color: #9F8F3C;
+  background-color: #A08F3C;
 }
 .ace_marker-layer .ace_find_line {
   position: absolute;
   z-index: -1;
-  background-color: #5F5F5F;
+  background-color: #606060;
 }
 .ace_console_error {
-  background-color: #5F5F5F;
+  background-color: #606060;
```

Comment:
Yes, part of this PR involved a change from `round` to `ceiling` because the output of `round` can change depending on the operating system. Actually, it's output somehow changed when I switched from `highlight::css.parser` to `.rs.parseCss`, despite that part of the code not using the css parser at all. I manually went through every result change, and in some cases did the math myself (for example if the difference was not simply +1).